<!DOCTYPE html>
<html>
    <head>
        <title> To Do: Design Wireframe</title>
      
    </head>
    <style>
        img{
            width:"500";
            height:"600";
        }
    </style>

    <body>
    
        <h1>Week 5 Notes: Testing Frameworks</h1>
        <br>
        <p>This week I did the test exercises from the book to understan how it works and how it is useful.</p>
        <p> I learn that errors and bugs are always there and we need to have the ability of minimize, identify and deal with them.</p>
        <p>I also learn how to use jest to test</p>
        <br>
        </p>
        <h3>Test 1</h3>
        <img src="testImg/test2.png">
        <br>
        <h3>Test 2</h3>
        <img src="testImg/test1.png">
        <br>
        <h3>Test 3</h3>
        <img src="testImg/test3.png">
        <br>
        <h3>Test 4</h3>
        <img src="testImg/test4.png">
        <br>
        <h3>Test 5</h3>
        <img src="testImg/test5.png">
        <br>
        <h3>Test 6</h3>
        <img src="testImg/test6.png">
        <br>
        <h3>Test 7</h3>
        <img src="testImg/test7.png">
        <br>
        <h3>Test 8</h3>
        <img src="testImg/test8.png">
        <br>
        <h3>Test 9</h3>
        <br>
        <p>The last test would have to pass all. I have reviewed the whole process and I can't find the reason why it doesn't throw the exception.
            Any ideas?</p>
        <img src="testImg/test9.png" >
        <br>

    
        
    
    </body>
</html>
    